### DISCUSSION TITLE
Wall Comment Deactivation Message

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#1050] [Ideas]
There seems to be a small issue with user pages of users who have user wall comments disabled. Usually at the bottom of a user page there is a header called "User Wall" and then under it is either the text "No comments." when there are none or the comments if there is some, additionally an input field to create a comment. If a user has the comments disabled for their page it seems to completely lack that section giving no information why it's missing.

If you are unaware that there is an option to disable it you likely think that the website doesn't load the page correctly because there is no information why it's not visible. My test alt where you can see this issue is here:
https://retroachievements.org/user/Highscore

![image](https://user-images.githubusercontent.com/45672834/178825520-dd2c937c-9011-4ae8-aec2-d731d7b1ee0c.png)
The image shows that right under the last game you can directly see the fodder of the page and it looks like something wasn't loaded.

I think some information that the comments were disabled would be helpful for people to know why it's gone. I am not a real coder myself, but here is some example I did with slightly HTML edits, text is just to give an idea, maybe the wording could be improved.
![image](https://user-images.githubusercontent.com/45672834/178827493-c61dbdd8-7f08-437c-8c04-762fa748ac94.png)

**Comment by pinguupinguu** [comment]
Great suggestion Spory.

I think the disable notification could even just say: "This user has disabled comments." 

It's implied by "User Wall" that they're wall comments. :)

**Comment by Jamiras** [comment]
implemented in #1059
